README.md

```diff
@@ -29,6 +29,9 @@
 - Plays nice with CI and source control
 - Run globally or locally as a standalone package app or `require('backstopjs')` right into your node app
 - Incredibly easy to use: just 3 commands go a long long way!
+- [BETA] Limited Support for WebDriver Protocol, based on wdio
+   - works as well with any wdio-(cloud)-service (see [wdio configuration]() )
+   - not compatible with puppeteer references pictures
```

Comment:
I had some issues with directly comparing, backstopjs puppeteer pictures with wdio pictures. 
- screensize is different a bit (still need to investigate if that what goes on, but due sickness&work I couldn't find the time) 
- puppeteer does a full page picture, webdriver captures only viewport => In theory, should be possible to merge the viewport images to a full page picture, wdio plugin is outdated
- image interception is not possible without a proxy solution, due selenium limitation 


Therefore I added a BETA support, so maybe some people get interessted to work on it.

Reply:
Oh, it is very very unlikely that you could ever render a reference image in puppeteer and then be able to test that against a test image generated in WDIO.  Or vice versa.   In addition to different high level rendering interpretations of the different application environments, there will also be a low-level rendering issue because different hardware will dither bitmap text differently.  [For example you can see this explanation](https://github.com/garris/BackstopJS#using-docker-for-testing-across-different-environments).    

TLDR;   Screenshots generated by a single hardware, software and platform combination can only be compared against itself.